### ISSUE TITLE
Enhancement: Add support for --target-cpu=native

### ISSUE THREAD

**Issue by itamarst** [mityax/rustimport#21]
For local development, or just fixed hardware, it would be nice to take advantage of the fact that one is not distributing the binaries but compiling them locally. Being able to add `--target-cpu=native` to RUSTFLAGS would make some code even faster.

This is particularly helpful in numeric code where you can get the compiler to use SIMD automatically, making `rustimport` a stronger alternative to Numba, which already takes advantage of current CPU features by default.

It's unclear whether this should be on by default. Certainly you want it off anytime you are building on one machine and running the resulting code an arbitrary different machine, because you might end up generating code that won't run on other CPUs with different instruction sets (e.g. not all x86-64 CPUs have AVX-512 or AVX2).

**Comment by mityax** [comment]
Yeah, that'd be nice. Since rustimport is using `subprocess.Popen` to invoke cargo, which makes it inherit the env your script is started with by default, it should work.
Also, it has already been working with `ARCHFLAGS` (see #23)
